src/sonic-yang-models/tests/yang_model_tests/yangTest.json

```diff
@@ -1300,267 +1326,419 @@
 				"lanes": "65",
 				"description": "",
 				"speed": "11100",
-				"admin_status": "up"
+				"admin_status": "up",
+				"autoneg": "1",
```

Comment:
Is it still 1 and 0 and not on and off?

Reply:
Still 1 and 0.

Reply:
we have discussed this in the yang community subgroup, the recommendation is to use on|off

Reply:
sure